### ISSUE TITLE
Feature Req: Make Repository multi-select for creation of duplicate issues across language repos

### ISSUE THREAD

**Issue by christothes** [AlexGhiondea/IssueCreator#37]
Would be nice to be able to create the same issue across multiple repos with a single click.

**Comment by AlexGhiondea** [comment]
I think we might be able to do this but we probably don't have a good way of assigning it across the repos.

We can probably make the drop-down for the repositories a selection list (you check which repos you want to create the issue). But if you do that you would lose the ability to assign to specific people and maybe even the ability to add the issues to an epic... unless you want to add all the issues to the same epic?

**Comment by christothes** [comment]
> I think we might be able to do this but we probably don't have a good way of assigning it across the repos.
> 
> We can probably make the drop-down for the repositories a selection list (you check which repos you want to create the issue). But if you do that you would lose the ability to assign to specific people and maybe even the ability to add the issues to an epic... unless you want to add all the issues to the same epic?

I think it would be ok to eliminate user assignment when doing a multi-select. For the epic, I think there are cases where assigning the same epic makes sense, but if not you wouldn't need to select one.

**Comment by AlexGhiondea** [comment]
I agree -- I think we can get this done in a way that is useful in the 80% case! 

I'll see if I can get this started once the big refactoring PR is in (#36)
